tools/gen_ort_dockerfile.py

```diff
@@ -283,7 +279,7 @@ def dockerfile_for_linux(output_file):
         )
         cuda_archs = "53;62;72;87"
```

Comment:
We probably still can try to compile for sm_101
```suggestion
        cuda_archs = "53;62;72;87;101"
```